test/common/test_common.cpp

```diff
@@ -360,27 +360,27 @@ TEST (PCL, SetIfFieldExists)
 //  pcl::for_each_type<pcl::traits::fieldList<pcl::PFHSignature125>::type> (pcl::SetIfFieldExists<pcl::PFHSignature125, float*> (p2, "intensity", 3.0));
 }
 
-//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////
 TEST (PCL, IsSamePointType)
 {
-  bool status = isSamePointType<PointXYZ, PointXYZ> ();
-  EXPECT_TRUE (status);
-  status = isSamePointType<PointXYZ, PointXY> ();
-  EXPECT_FALSE (status);
-  status = isSamePointType<PointXY, PointXYZ> ();
-  EXPECT_FALSE (status);
-  status = isSamePointType<PointNormal, PointNormal> ();
-  EXPECT_TRUE (status);
-  status = isSamePointType<PointNormal, PointXYZRGBNormal> ();
-  EXPECT_FALSE (status);
-  status = isSamePointType<PointXYZRGB, PointXYZRGB> ();
-  EXPECT_TRUE (status);
-
+  constexpr const bool status0 = isSamePointType<PointXYZ, PointXYZ> ();
+  EXPECT_TRUE (status0);
```

Comment:
Thoughts on combining into a single line?

Reply:
done
also, only two of these tests seem like relevant, should i remove duplication, eg there's no point to test
`EXPECT_TRUE (isSamePointType<PointNormal, PointNormal> ());`
if there's
`EXPECT_TRUE (isSamePointType<PointXYZ, PointXYZ> ());`
what do you think?

Reply:
hi! should i revert my changes in tests?

Reply:
Sorry for the delay.

> someone reading those tests would (mistakenly) assume that there is some difference in comparison logic for PointXYZ and PointNormal types

I don't really agree with that thought. I think it is normal to test a function with many different parameters, also template parameters, without any implication that the function works differently for some parameters. I think the `EXPECT_TRUE` and `EXPECT_FALSE` also make clear what the function should do in each case. If you want to add a clarifying comment somewhere, that is of course always a good idea. I just think removing already existing tests is counter-intuitive, considering that too much code in PCL is untested.